src/crimson/osd/shard_services.cc

```diff
@@ -808,7 +808,7 @@ seastar::future<MURef<MOSDMap>> OSDSingletonState::build_incremental_map_msg(
                           crimson::make_message<MOSDMap>(
                             monc.get_fsid(),
                             osdmap->get_encoding_features()),
-                          [this, &first, FNAME, last](unsigned int map_message_max,
+                          [this, &first, FNAME, last](auto &map_message_max,
```

Comment:
yeah, would be much simpler if we could use coroutine here..

Reply:
Soon!